src/models/rating.ts

```diff
@@ -0,0 +1,24 @@
+import mongoose, { Schema } from "mongoose";
+const systemRating = mongoose.model(
+    "systemRating",
+    new Schema({
+      name: {
+        type: String,
+        required: true,
+        unique: true,
+      },
+      grade: {
+        type:[Number],
+        required: true,
+      },
+      description: {
+        type:[String],
+        required:true,
+      },
+      percentage: {
+        type:[String],
+        required:true
+      },
+    })
+  );
```

Comment:
Can we add a field that ties a logged in user to the rating system when it is being created?

That way, we can allow the user to query the rating system that they have created